server/src/sim/viewresults.py

```diff
@@ -185,16 +185,20 @@ def viewmultiresults(M, whichgraphs={'prev':[1,1], 'plhiv':[0,1], 'inci':[0,1],
 
```

Comment:
I see that viewresults is changed, but makeresults isn't. Why is so? The web application doesn't plot graphs on the backend.